__tests__/controllers/auth.controller.spec.ts

```diff
@@ -1,7 +1,7 @@
 import request from 'supertest';
 import app from '@app';
 import connection from '@database/connection';
-import { API, mockUserFields } from '../utils/index';
+import { API } from '../utils/index';
 import { factory, useSeeding } from 'typeorm-seeding';
 import { User } from '@entities/user.entity';
```

Comment:
Just a quick question, why are we using `non-relative` imports with `@` instead of using a relative path `./` ? I think this is more clear for the team to know about the structure.

Reply:
I don't think reading something like `../../../../entities` would help anyone understand the structure. We have a readme for that and I think this is the most flat structure that we can have adding a separation of responsibilities. Perhaps we can help the team to know more about the structure with `how to` on the wiki or adding more info to the readme.

Reply:
make sense, we will need to turn off some `sonarqube` typescript-rules to avoid being marked this as a code smell.
<img width="1399" alt="Screen Shot 2020-09-23 at 12 58 48" src="https://user-images.githubusercontent.com/3788239/94038347-b1ec4b80-fd9c-11ea-9580-5530b2d1f659.png">

Reply:
Done!